components/index/cards/hackathons.js

```diff
@@ -229,25 +221,28 @@ export default function Hackathons({ data, stars }) {
                   justifyContent: 'center',
                   borderRadius: 'circle',
                   height: ['20px', '25px', '30px'],
-                  width: ['20px', '25px', '30px']
+                  width: ['20px', '25px', '30px'],
+                  mr: '5px'
                 }}
               >
                 <Image
                   src={data.logo}
-                  alt={`${data.name} logo`}
```

Comment:
pls keep this for accessibility reasons

Reply:
the text on the image is also written out as text (title of the hackathon) so i thought it might be repetitive? what do you think?

Reply:
screen readers use alt text for describing images 🙂 also its just semantic to have a good description as alttext